src/homepageExperience/components/steps/Tokens.tsx

```diff
@@ -79,14 +79,14 @@ export const Tokens: FC<OwnProps> = ({
       dispatch(createAuthorization(authorization))
       event(`firstMile.${wizardEventName}.tokens.tokenCreated`)
     }
-  }, [sortedPermissionTypes.length])
+  }, [sortedPermissionTypes.length, tokenValue]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
If you add `tokenValue`, you should need to add `// eslint-disable-line react-hooks/exhaustive-deps`. Is that to resolve another missing dependency - like `wizardEventName`? It's totally fine to include those in the dep array, they won't change.